packages/create-next-app/helpers/get-pkg-manager.ts

```diff
@@ -8,6 +8,8 @@ export function getPkgManager(): PackageManager {
       return 'yarn'
     } else if (userAgent.startsWith('pnpm')) {
       return 'pnpm'
+    } else if (userAgent.startsWith('bun')) {
```

Comment:
Not sure if this works because you don't install `bun` through npm (e.g. `npm_config_user_agent` from the var above).

Reply:
We do set the `npm_config_user_agent` when invoked via `bun run` or `bun`

https://github.com/oven-sh/bun/blob/3b259211df10e06ddf5844898a6ffa3c9679e4d8/src/cli/run_command.zig#L634-L644

Reply:
Nice!